src/settings.ts

```diff
@@ -520,34 +572,74 @@ export class YAxisLabelsSettings extends FormattingSettingsCard {
         name: "show",
         displayNameKey: "Visual_Show",
         value: true,
-        topLevelToggle: true, 
+        topLevelToggle: true,
     });
 
     public fill = new formattingSettings.ColorPicker({
         name: "fill",
         displayNameKey: "Visual_LabelsFill",
-        value: { value: "#aaa" }, 
+        value: { value: "#aaa" },
+    });
+
+    public fontControl = new formattingSettings.FontControl({
+        name: "fontControl",
+        displayNameKey: "Visual_FontControl",

```

Comment:
The display name is broken right now.
![image](https://github.com/microsoft/powerbi-visuals-heatmap/assets/94454577/dac4982d-fcb3-481b-b240-2ba9247b2770)


Every displayNameKey should be declared in all localization files (path is `./stringResources/*/**.resjson`). But we as developers responsible only for en-US translation. So, please, add `Visual_FontControl` and `Visual_Margin` in the appropriate file.

Reply:
Thanks for clarifying this. Here is the commit: [0613d7b](https://github.com/microsoft/powerbi-visuals-heatmap/pull/118/commits/0613d7b85609ca7a522d1417b4c7eefc41ee2a6e)